.github/workflows/test.yaml

```diff
@@ -1,33 +1,42 @@
 name: test
 on:
   pull_request:
-    workflow_dispatch:
+  workflow_dispatch:
```

Comment:
Do we need to preserve the indentation to have valid yaml?

Reply:
workflow_dispatch is one of the triggers, the previous indentation was incorrect. It should add 

<img width="960" alt="Screenshot 2024-06-09 at 4 08 13 PM" src="https://github.com/dev-launchers/strapiv4/assets/2912772/593807a0-eda6-492f-86f8-c3b00ccee8c8">